src/main/java/gov/usds/case_issues/CaseIssueApi.java

```diff
@@ -4,9 +4,11 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.retry.annotation.EnableRetry;
 
 @SpringBootApplication
 @EnableConfigurationProperties
+@EnableRetry
```

Comment:
Is it possible this should be on a configuration file that we only include if we are in fact doing retries? I would guess it's a pretty low-overhead thing to include if we're not using it, but still.